libs/SalesforceSDK/src/com/salesforce/androidsdk/config/LoginServerManager.java

```diff
@@ -375,9 +373,9 @@ private void persistLoginServer(String name, String url, boolean isCustom, Share
 		}
 		int numServers = sharedPrefs.getInt(NUMBER_OF_ENTRIES, 0);
 		final Editor edit = sharedPrefs.edit();
-		edit.putString(String.format(SERVER_NAME, numServers), name);
-		edit.putString(String.format(SERVER_URL, numServers), url);
-		edit.putBoolean(String.format(IS_CUSTOM, numServers), isCustom);
+		edit.putString(String.format(Locale.US, SERVER_NAME, numServers), name.trim());
```

Comment:
`trim()` while writing - for rogue custom URLs that were stored with whitespace in the current/future versions of the app.